centreon/packaging/debian/centreon-web.postinst

```diff
@@ -11,9 +11,7 @@ if [ "$1" = "configure" ] ; then
     chmod 0775 \
       /usr/share/centreon/src \
       /usr/share/centreon/api
-    chmod -R 0775 \
-      /var/log/centreon \
-      /var/lib/centreon
+    chmod -R 0775 /var/lib/centreon
```

Comment:
Why not 774 on/var/log/centreon like in the RPM spectemplate?

Reply:
`/var/log/centreon` is only a dir from `centreon-poller-centreon-engine` package
so 0774 must only be in `centreon-poller-centreon-engine.postinst`